CODEOWNERS

```diff
@@ -1,2 +1,4 @@
 # Main team administering overall repo
 *           @bharathkkb @g-awmalik @apeabody
+# ADC team reviewers
+/cli/ @q2w
```

Comment:
@bharathkkb the last match takes precedence and doesn't include the default.  Perhaps you want to also include https://github.com/orgs/GoogleCloudPlatform/teams/blueprint-solutions?

Reply:
Oh I thought the first took precedence. Added solutions team as well.